modules/client/jvm/src/test/scala/com/azavea/stac4s/api/client/StacClientSpec.scala

```diff
@@ -0,0 +1,130 @@
+package com.azavea.stac4s.api.client
+
+import com.azavea.stac4s.testing.JvmInstances
+
+import cats.effect.{Blocker, IO}
+import cats.syntax.applicative._
+import cats.syntax.either._
+import com.azavea.IOSpec
+import eu.timepit.refined.collection.Empty
+import eu.timepit.refined.types.all.NonEmptyString
+import io.circe.JsonObject
+import io.circe.syntax._
+import org.scalatest.BeforeAndAfterAll
+import sttp.client3.asynchttpclient.cats.AsyncHttpClientCatsBackend
+import sttp.client3.http4s.Http4sBackend
+import sttp.client3.impl.cats.CatsMonadAsyncError
+import sttp.client3.testing.SttpBackendStub
+import sttp.client3.{Response, UriContext}
+import sttp.model.Method
+
+class StacClientSpec extends IOSpec with JvmInstances with BeforeAndAfterAll {
+
+  lazy val backend =
+    SttpBackendStub(new CatsMonadAsyncError[IO]())
+      .whenRequestMatches(_.uri.path == Seq("search"))
+      .thenRespondF { _ =>
+        Response
+          .ok(arbItemCollectionShort.arbitrary.sample.asJson.asRight)
```

Comment:
Because this is only generating two items, I don't think it's exercising the things I mentioned above about the pagination token

Reply:
Yea, we don't handle pagination at all. I will create an issue to cover it (I think it is already in the gt server repo).

Reply:
Opened an issue for that https://github.com/azavea/stac4s/issues/198